content/articles/email-forwarding.md

```diff
@@ -20,31 +20,32 @@ The number of forwarded messages and email forwards allowed per month is based o
 
 ## Enabling email forwarding
 
-To enable email forwarding, visit a domain's manage page and click on the **Email forwarding** menu on the sidebar.
+To enable email forwarding, 
 
-![Email forwarding](/files/email-forwarding.png)
+1. Choose the relevant domain from your Domain List.
+2. Click the **Email Forwarding** tab menu on left side. 
 
-Create an email forward to enable the service.
+<!--- needs screenshot -->
```

Comment:
https://drive.google.com/file/d/1BAC7FbzAA7tGSIz197qzVP34YnVDzEbt/view?usp=sharing

Reply:
The screenshot shows the "Connections" section. Am I looking wrong?

Reply:
yep that's the wrong one will replace